src/lib/mina_compile_config/mina_compile_config.ml

```diff
@@ -67,7 +67,7 @@ let max_event_elements = 100
 
 let max_action_elements = 100
 
-[%%inject "network_id", network]
+let network_id = "testnet"
```

Comment:
Wouldn't this impact when building from source using mainnet profile?

Reply:
Yes, but only locally. In CI, this is still true. It's also only used in one janky GraphQL endpoint anyway, where it's overridden by the `ledger.name` in the runtime config.

Reply:
It seems like there are members in the community that build from source and not use the deb or docker images from releases. So we'll need to keep that option available

Reply:
>It's also only used in one janky GraphQL endpoint anyway, where it's overridden by the ledger.name in the runtime config.

oh :/